src/OrderLinesList/OrderLinesList.test.js

```diff
@@ -24,6 +24,9 @@ const mockLocalStorageFilters = {
 jest.mock('react-virtualized-auto-sizer', () => jest.fn(
   (props) => <div>{props.children({ width: 123 })}</div>,
 ));
+jest.mock('@folio/stripes-core/src/components/TitleManager/TitleManager', () => {
```

Comment:
It should be mocked on the `stripes-acq-components` level in jest config. Please try to update deps and run tests without this code

Reply:
Like that? https://github.com/folio-org/stripes-acq-components/commit/0686ef186079840b960cc85997f69b90673954df

Reply:
Yes